pkg/kubelet/kubelet_network_linux.go

```diff
@@ -47,14 +47,16 @@ func (kl *Kubelet) initNetworkUtil() {
 		iptClients = append(iptClients, utiliptables.New(exec, utiliptables.ProtocolIPv6))
 	}
 
-	for _, iptClient := range iptClients {
-		kl.syncNetworkUtil(iptClient)
-		go iptClient.Monitor(
-			utiliptables.Chain("KUBE-KUBELET-CANARY"),
-			[]utiliptables.Table{utiliptables.TableMangle, utiliptables.TableNAT, utiliptables.TableFilter},
-			func() { kl.syncNetworkUtil(iptClient) },
-			1*time.Minute, wait.NeverStop,
-		)
+	for i := range iptClients {
+		iptClient := iptClients[i]
+		if kl.syncNetworkUtil(iptClient) {
+			go iptClient.Monitor(
+				utiliptables.Chain("KUBE-KUBELET-CANARY"),
+				[]utiliptables.Table{utiliptables.TableMangle, utiliptables.TableNAT, utiliptables.TableFilter},
+				func() { kl.syncNetworkUtil(iptClient) },
+				1*time.Minute, wait.NeverStop,
+			)
+		}
```

Comment:
I think we should log a warning to inform that we are not going to monitor this family, it is possible that the admin didn't know why is not working, so observing the log at least he has a clue and can install the missing kernel modules, per example, and restart kubelet